package.json

```diff
@@ -22,7 +23,8 @@
     "build:lyra": "cd packages/lyra && pnpm build",
     "build:cjs": "tsc --project tsconfig.cjs.json",
     "build:module": "tsc --project tsconfig.esm.json",
-    "build": "rimraf ./dist && pnpm build:module && pnpm build:cjs",
+    "build:webpack": "tsc --project tsconfig.webpack.json",
```

Comment:
Are we planning to use `webpack` for something different than React Native (i.e., browser bundles)? Otherwise I'd change this to `build:react-native`

Reply:
No really not, but I think indirectly it will be picked up by anyone using next.js (I can't find docs on this) or create-react-app (uses webpack 5)... that's why I was trying to keep it generic. maybe renaming it to `browser` everywhere would be more intuitive??? What do you think?

Reply:
I tested this in Next.js and it works well (Next.js uses Webpack 5)!

In any case, we can go with browser and make it more generic 👍